enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,223 @@
-const enhancer = require('./enhancer.js');
+const { repair, succeed, fail, get } = require("./enhancer.js");
 // test away!
+// test suite
+describe("the game Succeed/Fail/Repair module", () => {
+  // test suite
+
+  // REPAIR MODEL TEST
+  describe("the repair module", () => {
+    // test
```

Comment:
Note that you are able to define globals; for example, you could have a single `improperObject` for your test cases that use the same one. So long as it's outside of their scope (in the scope above theirs), it'll be usable for all of them.

Reply:
Try to apply this same thing to other places where you use the same test data; if you have an impure function, then you could always use a function to generate the data object rather than just the data object.